explainer.md

```diff
@@ -0,0 +1,107 @@
+# Screen Wake Lock API
+
+## Introduction
+The **Screen Wake Lock API** allows web applications to prevent a device's
+screen from dimming and locking.
+
+See also:
+* [The current specification](https://w3c.github.io/screen-wake-lock/)
+* [web.dev article](https://web.dev/wakelock/) on the Screen Wake Lock API
+
+## Why?
+To avoid draining the battery, most devices quickly go to sleep when left idle.
+While this is fine most of the time, some applications need to keep the screen
+awake to complete their work. For example:
+
+* A recipe app that keeps the screen on while you bake a cake or cook dinner.
+* A boarding pass or ticket app that keeps the screen on until the barcode has
+  been scanned.
+* A kiosk-style app that keeps the screen on continuously.
+* A web-based presentation app that keeps the screen on during a presentation.
+
+Without this API, web developers have had to rely on libraries such as
+[NoSleep.js](https://github.com/richtr/NoSleep.js), which essentially added a
+[small
+video](https://github.com/richtr/NoSleep.js/blob/eaf52afd1dfbb80145b4a39f3ec29307b80ab154/src/index.js#L37-L57)
+to a page to keep the display on.
+
+> Recent NoSleep.js releases have added support for the Screen Wake Lock API.
+
+The purpose of this specification is to allow sites to be explicit about when
+they want to acquire a wake lock so that browsers can provide better automated
+and manual controls to prevent intentional and incidental issues.
+
+## Non-goals
+A previous iteration of this spec included both Screen and System wake locks.
+The latter has been moved to a separate spec, and is out of scope for this
+explainer.
+
+## API
+The API surface is not big: users can request a screen wake lock; if the
+request succeeds it can be either manually released later or released
+automatically by the platform due to OS-specific policies that might be in
+place. The code looks like this:
+
+```js
+const lock = await navigator.wakeLock.request("screen");
+
+lock.addEventListener("release", () => {
+  console.log(`Has the lock been released? ${lock.released}`);
+});
+await lock.release();
+```
+
+### WakeLock
+The `WakeLock` object is part of `Navigator`. It has a single method,
+`request(type)`. The `type` argument is of the `WakeLockType` enum type, which
+right now has a single value, _"screen"_.
+
+`WakeLock.request()` performs permission checks, verifies that the document
+issuing the request is visible and, if all checks succeed, asynchronously
+requests a lock from the underlying platform to prevent the screen from turning
+off.
+
+A successful `WakeLock.request()` invication returns a promise that will
+ultimately resolve with a `WakeLockSentinel` object.
+
```

Comment:
The statement "The `WakeLock` object **is part of** `Navigator`" doesn't make much sense, tbh. Nothing is actually "part of" Navigator.  It doesn't really make sense to talk about the `WakeLock` and `Navigator` interfaces at all, but only how they are access. 

Additionally, this goes into details that are only really relevant for browser engineers (e.g., enum values)

```suggestion
### Requesting permission to use WakeLocks
To use the API, one must request access via `navigator.wakeLock.request("screen")`. This returns a promise which, if allowed by the user, resolves with a `WakeLockSentinel`. 

```JS
let sentinel;

try {
  sentinel = await navigator.wakeLock.request("screen")
} catch (err) {
  // we got denied access or something went wrong...
}
```

```

Reply:
Done.